### PR TITLE
Dropdowns rtl issues and invalid state styling

### PR DESCRIPTION
**Pull request by inikolova**
Related to https://github.com/telerik/kendo-theme-default/issues/493

### CODE REVIEW

**Review by gyoshev — CHANGES_REQUESTED** [review]
Some things we discussed offline:

1. We should remove the rules that mention `&:invalid:not(.ng-valid)`, as these get applied in Kendo UI for jQuery by default, and do not look good
2. The invalid styling can be moved to `common/forms` and get cascaded through `.k-widget.ng-invalid`
3. The `$invalid-border-color` can perhaps be moved to `_variables.scss`.

**Comment by inikolova** [comment]
The discussed changes have been made

**Review by gyoshev — APPROVED** [review]

File: scss/common/_forms.scss [hunk 1]
```diff
@@ -225,6 +225,20 @@
...
+    .k-widget.ng-invalid {
+        .k-dateinput-wrap,
+        .k-dropdown-wrap,
+        .k-numeric-wrap,
+        .k-picker-wrap {
+            color: $error;
+            border-color: $invalid-border-color;
+
+            .k-select {
+                color: $error;
```
Comment (joneff) [inline scss/common/_forms.scss]: if the color is the same, shouldn't `color: inherit` do the same thing?

**Review by joneff — APPROVED** [review]

File: scss/common/_forms.scss (hunk 1 above)
Comment (inikolova) [reply scss/common/_forms.scss]: @joneff , good catch! I will make this change

**Pull request merged**